python/pyarrow/tests/strategies.py

```diff
@@ -46,14 +46,31 @@
 import pyarrow as pa
 
 
-# TODO(kszucs): alphanum_text, surrogate_text
+# Text generation strategies for various character sets:
 custom_text = st.text(
     alphabet=st.characters(
         min_codepoint=0x41,
         max_codepoint=0x7E
     )
 )
 
```

Comment:
I referred to https://www.unicode.org/reports/tr44/#General_Category_Values and https://datatracker.ietf.org/doc/html/rfc2781#section-2.1